src/linkify.js

```diff
@@ -0,0 +1,54 @@
+import * as scanner from './linkify/core/scanner';
+import * as parser from './linkify/core/parser';
+
+/**
+	Converts a string into tokens that represent linkable and non-linkable bits
+	@method tokenize
+	@param {String} str
+	@return {Array} tokens
+*/
+let tokenize = function (str) {
+	return parser.run(scanner.run(str));
+};
+
+/**
+	Returns a list of linkable items in the given string.
+*/
+let find = function (str) {
```

Comment:
Let's add an optional `type` param here as well so we can find all the links of a certain type